components/crypto/crypto-service-impl/src/main/kotlin/net/corda/crypto/service/impl/bus/CryptoOpsBusServiceImpl.kt

```diff
@@ -76,15 +78,18 @@ class CryptoOpsBusServiceImpl @Activate constructor(
         logger.info("Creating RPC subscription for '{}' topic", Schemas.Crypto.RPC_OPS_MESSAGE_TOPIC)
         val messagingConfig = event.config.getConfig(MESSAGING_CONFIG)
         val processor = CryptoOpsBusProcessor(
-            event = event,
-            cryptoServiceFactory = cryptoServiceFactory,
-            schemeMetadata = schemeMetadata,
-            digestService = digestService,
-            keyEncodingService = keyEncodingService,
-            dbConnectionManager = dbConnectionManager,
-            jpaEntitiesRegistry = jpaEntitiesRegistry,
-            virtualNodeInfoReadService = virtualNodeInfoReadService,
-            layeredPropertyMapFactory = layeredPropertyMapFactory
+            SigningServiceImpl(
+                cryptoServiceFactory = cryptoServiceFactory,
+                schemeMetadata = schemeMetadata,
+                digestService = digestService,
+                keyEncodingService = keyEncodingService,
+                dbConnectionManager = dbConnectionManager,
+                jpaEntitiesRegistry = jpaEntitiesRegistry,
+                virtualNodeInfoReadService = virtualNodeInfoReadService,
+                layeredPropertyMapFactory = layeredPropertyMapFactory,
+                config = event.config.get(CRYPTO_CONFIG)
+                    ?: throw InvalidParameterException("$CRYPTO_CONFIG missing from config")
```

Comment:
There's a conveience function for this.
```suggestion
               config = event.config.toCryptoConfig()
```

Different exception but...